src/scss/_typography.scss

```diff
@@ -21,9 +21,12 @@ $headings-font-weight: 600;
 $large-headings-font-weight: 700;
 
 $h1-font-size: $font-size-base * 3.125;
+// $h1-font-size: $font-size-base * 3.1875;
```

Comment:
Recuerdo que me comentaste esto, pero no el por qué.
¿Era un cambio general en la definición de todas las tipografías?

Reply:
Me fijé el tamaño del <h4> que tiene Obelisco actualmente y vi que es de 20px (1.25rem) cuando en el UI Kit está defindo en 21px (1.3125rem). Repasé el resto y en los que noté diferencias dejé las correcciones comentadas para avisarte y que fuera revisado.

Reply:
Ok, puede ser que cuando lo armé terminé redondeando a números más cerrados para que también se ajusten a los tamaños de grillas.

Por ahora saquémoslo de este PR y armemos otro con esos cambios que se llame "Ajustes de tipografía" o algo así, ¿te parece?